src/controllers/authenticationController.ts

```diff
@@ -104,6 +106,14 @@ export default class AuthenticationController {
 
     const spidUser = errorOrSpidUser.value;
 
+    if (!isOlderThan(14)(parse(spidUser.dateOfBirth), new Date())) {
```

Comment:
Since this value refers to a specific behaviour, maybe we should externalize this property into an env variable or even into a commented `const`.

Reply:
Fixed in 06d1104